CentralityiOS/CentralityiOS/ViewControllers/CentralityHelpers.m

```diff
@@ -39,6 +39,15 @@ + (void) updateLabel:(UILabel*)label newText:(NSString*)newText isHidden:(BOOL)i
     label.hidden = isHidden;
 }
 
++ (void)transitionLabel :(UILabel*)label newText:(NSString*)newText{
+    [UILabel transitionWithView:label
+                       duration:0.25f
+                        options:UIViewAnimationOptionTransitionCrossDissolve
+                     animations:^{
+        label.text = newText;
```

Comment:
NIT: this is personal preference and sort of up to whatever code linter you're working with, but i like to keep the `{` and `}` somewhat lined up. having the animations close bracket so far to the left makes it less obvious what's going on.